### PR TITLE
[13.0][FIX] maintenance_plan: add rule migration

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Oversight of migration to v13.

### CODE REVIEW

**Review by LoisRForgeFlow — APPROVED** [review]

**Review by etobella — APPROVED** [review]
Seem right. Just one question. It might have sense to add this on a new version change, just for people that already did the migration. WDYT?

**Review by etobella — APPROVED** [review]

**Comment by MiquelRForgeFlow** [comment]
Yeah, makes sense. It's done.

File: maintenance_plan/migrations/13.0.1.2.4/post-migration.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Copyright 2020 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+# pylint: disable=C7902
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.load_data(
+        env.cr, "maintenance_plan", "migrations/13.0.1.2.0/noupdate_changes.xml"
```
Comment (LoisRForgeFlow) [inline maintenance_plan/migrations/13.0.1.2.4/post-migration.py:10]: You need to change also the filename here!

**Review by LoisRForgeFlow — COMMENTED** [review]
This fails in any update with latest code

File: maintenance_plan/migrations/13.0.1.2.4/post-migration.py (hunk 1 above)
Comment (MiquelRForgeFlow) [reply maintenance_plan/migrations/13.0.1.2.4/post-migration.py:10]: https://github.com/OCA/maintenance/pull/210

**Pull request merged**